website/src/components/markdown-layout.js

```diff
@@ -13,7 +13,8 @@ const TypographyPage = styled.div`
   font-family: 'Raleway', sans-serif;
   font-weight: 400;
   line-height: 1.45;
-  padding: ${customProperties.spacingM};
+  padding: ${customProperties.spacingL};
+  max-width: 740px;
```

Comment:
This ensures that the content stays within a reasonable size. This width is the same of what Medium uses